<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginFrame</class>
 <widget class="QDialog" name="loginFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1949</width>
    <height>1049</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1951</width>
     <height>1061</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1951</width>
      <height>1071</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>student and guest.png</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>1761</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
border-radius: 20px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>70</y>
       <width>1001</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Montserrat&quot;;
color: rgb(21, 31, 41);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Student Appointment Form</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbInstructor">
     <property name="geometry">
      <rect>
       <x>1350</x>
       <y>330</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(21, 31, 41);
background-color: rgb(231, 231, 231);
font: 14pt &quot;Montserrat&quot;;
border-radius:-20px solid gray;</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>--Select Instructor--</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbDepartment">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>330</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(21, 31, 41);
background-color: rgb(231, 231, 231);
font: 14pt &quot;Montserrat&quot;;
border-radius:-20px solid gray;</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>--Select Department--</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BSCS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BSIT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BSIS</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>330</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 14pt &quot;montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>Set Appointment to:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAppointmentSubject">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>230</y>
       <width>701</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid gray;
color: rgb(21, 31, 41);
font: 14pt &quot;Montserrat&quot;;
border-radius: 5px;</string>
     </property>
     <property name="placeholderText">
      <string>Appointment Subject</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textDetails">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>430</y>
       <width>701</width>
       <height>211</height>
      </rect>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid gray;
font: 12pt &quot;Montserrat&quot;;
</string>
     </property>
     <property name="documentTitle">
      <string/>
     </property>
     <property name="placeholderText">
      <string>(Optional)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>380</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 14pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>Details:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntReset">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>670</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 74, 173);
	background-color: rgb(254, 206, 47);
font: 14pt &quot;Montserrat&quot;;
border-radius: 20px
}

</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSet">
     <property name="geometry">
      <rect>
       <x>1370</x>
       <y>670</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(255, 255, 255);
	background-color: rgb(0, 74, 173);
font: 14pt &quot;Montserrat&quot;;
border-radius: 20px;
}

;</string>
     </property>
     <property name="text">
      <string>Set Appointment</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>621</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(0, 74, 173);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font: 30pt &quot;Montserrat&quot;;
color: rgb(21, 31, 41);
background-color: rgb(255, 255, 255);
border-radisu: -20px
}

</string>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="widgetAdmin3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>601</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>INSTRUCTOR</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DEPARTMENT</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DATE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QLabel" name="lblTimeAndDate">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>160</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 14pt &quot;montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>886</x>
       <y>159</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 14pt &quot;montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>Date and Time:</string>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>label_6</zorder>
    <zorder>label_2</zorder>
    <zorder>cmbInstructor</zorder>
    <zorder>cmbDepartment</zorder>
    <zorder>inputAppointmentSubject</zorder>
    <zorder>textDetails</zorder>
    <zorder>bntReset</zorder>
    <zorder>btnSet</zorder>
    <zorder>frame_4</zorder>
    <zorder>lblTimeAndDate</zorder>
    <zorder>label_8</zorder>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>170</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnStudentLogout">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(21, 31, 41);
background-color: rgb(255, 255, 255);
border-radius: 40px;
font: 14pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>170</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnStudentSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(21, 31, 41);
background-color: rgb(255, 255, 255);
border-radius: 40px;
font: 14pt &quot;Montserrat&quot;;</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inputAppointmentSubject</tabstop>
  <tabstop>cmbDepartment</tabstop>
  <tabstop>cmbInstructor</tabstop>
  <tabstop>textDetails</tabstop>
  <tabstop>btnSet</tabstop>
  <tabstop>bntReset</tabstop>
  <tabstop>btnStudentSettings</tabstop>
  <tabstop>btnStudentLogout</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
